src/main/java/bookbob/functions/FileHandler.java

```diff
@@ -0,0 +1,159 @@
+package bookbob.functions;
+
+import bookbob.entity.Patient;
+import bookbob.entity.Records;
+import bookbob.entity.AppointmentRecord;
+import bookbob.entity.Appointment;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FileHandler {
```

Comment:
Maybe you could use slightly different method names to name the different methods in this file. I was confused for a bit when I saw multiple pairs of methods with the exact same method name.